tests/search/vespa/test_vespa_search.py

```diff
@@ -230,3 +232,44 @@ def test_search_with_deleted_docs(
     all_deleted_count = len(all_deleted_body["families"])
     assert start_family_count > one_deleted_count > all_deleted_count
     assert len(all_deleted_body["families"]) == 0
+
+
+@pytest.mark.search
+@pytest.mark.parametrize(
+    ("corpus_import_id", "corpus_type_name"),
+    [
+        ("CCLW.corpus.1.0", "UNFCCC Submissions"),
+        ("CCLW.corpus.1.0", None),
+        (None, "UNFCCC Submissions"),
+    ],
+)
+def test_corpus_filtering(
+    test_vespa,
+    monkeypatch,
+    data_client,
+    data_db,
+    valid_token,
+    corpus_import_id: str,
+    corpus_type_name: str,
+):
+    monkeypatch.setattr(search, "_VESPA_CONNECTION", test_vespa)
+    _populate_db_families(data_db)
+
+    params: dict[str, Any] = {"query_string": "and"}
+    if corpus_import_id:
```

Comment:
Why not just parameterise these and add `[]` or `None` when not needed?

Reply:
This is because it's different behaviour to have the field with it empty, then to have the field being absent